bugbug/model.py

```diff
@@ -104,7 +109,7 @@ def print_labeled_confusion_matrix(confusion_matrix, labels, is_multilabel=False
             table_labels = labels
 
         confusion_matrix_header = []
-        for i in range(len(table)):
+        for i in range(len(table[0])):
```

Comment:
Why this change?

Reply:
This is because the table has different sizes of rows and columns,earlier they're same.